core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -388,6 +388,13 @@ oppia.controller('ExplorationEditor', [
         '</ul>')
     }];
 
+    // Remove save from tutorial if user does not has edit rights for
+    // exploration since in that case Save Draft button will not be visible
+    // on the create page.
+    if (!GLOBALS.can_edit) {
+      $scope.EDITOR_TUTORIAL_OPTIONS.splice(9, 1);
```

Comment:
This is fragile ... if we added one more step to the tutorial before index 9 then that would throw this off and it's easy for that not to be detected. Any thoughts on how to address this?

Reply:
I have made a variable saveButtonElement and used that. PTAL!